e2e/chat.spec.ts

```diff
@@ -9,6 +9,9 @@ describe('chat', () => {
     const chat = new Chat(page);
     await chat.goto();
 
+    //  todo - remove when the default (reasoning) model is functioning
+    await chat.switchToBlockchainModel();
+
```

Comment:
We need to switch to the blockchain model (for now) for all e2e tests, because deepseek isn't wired in yet.

Once that happens, we can remove these from the simple text reply tests.